syncd/Syncd.cpp

```diff
@@ -4472,6 +4472,11 @@ static void timerWatchdogCallback(
     SWSS_LOG_ERROR("main loop execution exceeded %ld ms", span/1000);
 }
 
+#ifdef MDIO_ACCESS_USE_NPU
+extern int startIpcMdioProcessingThread(sai_switch_api_t *sai_switch_api);
+extern void stopIpcMdioProcessingThread(void);
+#endif
+
```

Comment:
move this to separate file
also why those 2 functions needs to be explicitly called instead on internals SAI when calling create_switch ?